docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+services:
+  rdmass:
+    container_name: rdmass
+    build:
+      context: ./rdmass
+    restart: unless-stopped
+    volumes:
+      - /etc/timezone:/etc/timezone:ro
+      - /etc/localtime:/etc/localtime:ro
+      - ./rdmass/config.json:/usr/src/app/config.json

```

Comment:
Would remove that relative path rdmass... This is used from you because you are using a shared yml for several services/tools I think. But for repo as standalone the path would be wrong then :)

Reply:
maybe I said it wrong, replace it with `- ./config.json:/usr/src/app/config.json`. I think that's better solution to match the general use-case :)